frontends/web/src/components/status/status.tsx

```diff
@@ -15,98 +15,77 @@
  * limitations under the License.
  */
 
-import { Component } from 'react';
-import { apiGet, apiPost } from '../../utils/request';
+import { ReactNode, useCallback, useEffect, useState } from 'react';
 import { CloseXWhite } from '../icon';
 import style from './status.module.css';
+import { getConfig } from '../../api/backend';
+import { setConfig } from '../../api/backend';
+
 
-interface State {
-    show: boolean;
-}
 
-interface StatusProps {
+type TPRops = {
     hidden?: boolean;
     type?: 'success' | 'warning' | 'info';
     // used as keyName in the config if dismissing the status should be persisted, so it is not
-    // shown again. Use an empty string if it should be dismissable without storing it in the
+    // shown again. Use an empty string if it should be dismissible without storing it in the
     // config, so the status will be shown again the next time.
-    dismissable?: string;
+    dismissible?: string;
     className?: string;
+    children: ReactNode;
 }
 
-type Props = StatusProps;
-
-export default class Status extends Component<Props, State> {
-  public readonly state: State = {
-    show: true,
-  };
-
-  public componentDidMount() {
-    this.checkConfig();
-  }
+const Status = ({
+  hidden,
+  type = 'warning',
+  dismissible,
+  className,
+  children,
+}: TPRops) => {
+  const [show, setShow] = useState(true);
 
-  public componentDidUpdate(prevProps: Props) {
-    if (this.props.dismissable !== prevProps.dismissable) {
-      this.checkConfig();
+  const checkConfig = useCallback(async () => {
+    if (dismissible) {
+      const config = await getConfig();
+      setShow(!config ? true : !config[dismissible]);
```

Comment:
they property is stored in `config.frontend[dismissible]`

best test with downgrading app version in update.go and restart `make servewallet`

```diff
diff --git a/backend/update.go b/backend/update.go
index 8b3b7b1d2..2162f72ad 100644
--- a/backend/update.go
+++ b/backend/update.go
@@ -27,7 +27,7 @@ const updateFileURL = "https://shiftcrypto.ch/updates/desktop.json"
 
 var (
        // Version of the backend as displayed to the user.
-       Version = semver.NewSemVer(4, 37, 0)
+       Version = semver.NewSemVer(4, 34, 0)
 )
 
 // UpdateFile is retrieved from the server.
```

also when dismissing the status banner, you should see an entry in appfolder.dev/config.json

Reply:
> they property is stored in config.frontend[dismissible]

Ups, yeah. 🤦  Shouldve been `config.frontend...`. My bad. thx!

Reply:
made the change and rebased.